src/components/elementView/detailedView/assetView/assetView.tsx

```diff
@@ -362,6 +365,45 @@ export class AssetView extends React.PureComponent<
             />
           </div>
         ) : null}
+
+        {this.state.asset.model &&
+        this.state.asset.model.model_type !== MountTypes.RACKMOUNT ? (
+          <div>
+            {this.state.asset.model.model_type === MountTypes.BLADE &&
+            this.state.asset.chassis ? (
+              <AnchorButton
+                disabled={
+                  !isNullOrUndefined(this.state.asset.decommissioning_user)
+                }
+                onClick={() =>
+                  this.props.history.push(
+                    ROUTES.ASSETS + "/" + this.state.asset.chassis!.id
+                  )
+                }
+                className="model-detail"
+                minimal
+                icon={IconNames.DOCUMENT_OPEN}
+                text="Go to chassis detail page"
```

Comment:
having weird behavior where the route updates correctly to the other asset, but the asset & model properties do not refresh. to create, I'm switching between bmi-chassis-1 and bmi-blade-1